src/components/MovieDetails.js

```diff
@@ -82,8 +88,8 @@ class MovieDetails extends Component {
           <h3 className='movieTitle'>{movie.title}</h3>
           <h3 className='rating'>{movie.average_rating.toFixed(1)}★</h3>
         </div>
-        {(this.state.user && !this.state.isFavorite) && <button className='favorite-btn' onClick={this.addFavorite}>Add to Favorites</button>}
-        {this.state.isFavorite && <button className='favorite-btn'>Remove from Favorites</button>}
+        {(this.state.user && !this.state.isFavorite) && <button className='favorite-btn' onClick={() => this.alterFavorites('add')}>Add to Favorites</button>}
+        {this.state.isFavorite && <button className='favorite-btn' onClick={() => this.alterFavorites('remove')}>Remove from Favorites</button>}
```

Comment:
Love using anonymous functions to pass in a method or function, I think the process of how that works is fascinating